### DISCUSSION TITLE
Images are not loading

### DISCUSSION THREAD

**Discussion by codingwithrk** [filamentphp/filament#8414] [General]
Hi, in https://filamentphp.com/ website docs the example images are not loading

**Comment by leandrocfe** [comment]
where? what page?

**Reply by codingwithrk** [reply]
Yesterday and before yesterday all examples are not loaded,  today all are loading

**Reply by leandrocfe** [reply]
weird, not sure if it was my browser cache, but I've used Filament Docs extensively, and it rarely breaks

**Comment by saleh-movahedi** [comment]
I had this problem on plugin page 🤦🏻‍♂️

**Reply by codingwithrk** [reply]
@saleh-movahedi Bro, I think it depends on the wifi because in my room wifi, I am facing this issue but with my own mobile network it was opening and also i tried with another wifi they are loading. I don't know why it was happening!

**Reply by saleh-movahedi** [reply]
So, it can be said that the problem is with the network, not necessarily Wi-Fi :smile:
